src/types/api-clients/amazon-selling-partner-api-base-path.ts

```diff
@@ -1,7 +1,7 @@
-export const enum AmazonSellingPartnerApiBasePath {
-  NA = 'https://sellingpartnerapi-na.amazon.com',
-  EU = 'https://sellingpartnerapi-eu.amazon.com',
-  FE = 'https://sellingpartnerapi-fe.amazon.com',
-}
+export type AmazonSellingPartnerApiBasePath =
+  | 'https://sellingpartnerapi-na.amazon.com'
+  | 'https://sellingpartnerapi-eu.amazon.com'
+  | 'https://sellingpartnerapi-fe.amazon.com'
```

Comment:
But this again limits us to a restricted set of inputs. Users should be allowed to specify any URL they want. That way they can pass sandbox URL explicitly, and we don't even need the sandbox flag. That also allows us to pass any URLs, e.g. for mocking and testing. And if Amazon adds another URL, nothing will break.